python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -916,6 +916,22 @@ def aten〇index〇Tensor(self: List[int], indices: List[Optional[List[int]]]) -
 def aten〇cat(tensors: List[List[int]], dim: int = 0) -> List[int]:
     return upstream_shape_helpers.cat(tensors, dim)
 
+def aten〇split(self: List[int], split_sizes: List[int], dim: int = 0) -> List[List[int]]:
```

Comment:
These should be contributed upstream. See https://github.com/llvm/torch-mlir/pull/908

Reply:
Added https://github.com/pytorch/pytorch/pull/79194. Please review. Thanks!